tools/docker/compose

```diff
@@ -4,7 +4,7 @@ set -ex
 export DOCKER_BUILDKIT=1
 export COMPOSE_DOCKER_CLI_BUILD=1
 
-base_files="-f docker-compose.yaml -f docker-compose.postgres.yaml"
+base_files="-f docker-compose.yaml -f docker-compose.postgres.yaml -f docker-compose.solana.yaml"
```

Comment:
We already have a custom logic based on $GETH_MODE, would it make sense to have the same for solana?

Reply:
It looks like `$GETH_MODE` toggle between go-ethereum vs. parity clients. AFAIK we don't have client variety on Solana, so we don't need an equivalent toggle.

Reply:
Sorry, I meant what happens if we want to run the node without Solana at all. Is the configuration supposed to be flexible?

Reply:
We could make it flexible. This setup predates multi-chain and non-evm support, and hasn't been updated with those in mind until now, so this is an open question.

Reply:
At the time, I just wanted a simpler way to test LOOP Plugins like Solana, without running CI integration tests.